### ISSUE TITLE
flask example log in button disappears 

### ISSUE THREAD

**Issue by dragonfly90** [pythonforfacebook/facebook-sdk#247]
fb-login-button seems not working. there is one error in safari 
```
TypeError: null is not an object (evaluating 'document.getElementById('logout').onclick = function() {
           FB.logout(function(response) { });
        }')
```
same in chrome

**Comment by martey** [comment]
Can you give more details? Is the button disappearing when you click it to login? Does the same error happen in non-Webkit browsers (e.g. Firefox, Internet Explorer)?

**Comment by dragonfly90** [comment]
I tried firefox, it still doesn't work. There is no Login button shown under flask example for chrome, firefox and safari.

**Comment by dragonfly90** [comment]
@martey,  I found that I should put the following code in index.html instead of base.html. Because the base.html should not have log out button. That fixation works. 
```
<script>
    document.getElementById('logout').onclick = function() {
        FB.logout(function(response) { });
    };
</script>
```
Secondly, in view.py, when we get user profile from GraphAPI, we should consider that someone may not have link property like me. So we should add an if statement in get_current_user function.

Hope my solution is right, and I make a pull request.

**Comment by martey** [comment]
I think this was closed because the issue is fixed by #259. If this isn't the case, let me know.
